src/main/java/mods/eln/gui/GuiContainerEln.java

```diff
@@ -159,29 +157,30 @@ protected void drawGuiContainerForegroundLayer(int mx, int my) {
 
         for (Object o : inventorySlots.inventorySlots) {
             Slot slot = (Slot) o;
-            if (!slot.getHasStack()
-                && mx - guiLeft >= slot.xPos && my - guiTop >= slot.yPos
-                && mx - guiLeft < slot.xPos + 17 && my - guiTop < slot.yPos + 17) {
+            if (slot.getHasStack() == false
```

Comment:
This can be simplified to `if(!slot.getHasStack())` and is much easier to understand.